Source/svg/SVGParser.swift

```diff
@@ -1433,8 +1433,12 @@ open class SVGParser {
         if let clipPath = attributes["clip-path"], let id = parseIdFromUrl(clipPath), let locus = locus {
             if let userSpaceLocus = defClip[id] {
                 if !userSpaceLocus.userSpace {
-                    let boundingBox = locus.bounds()
-                    let transform = ContentLayout.of(contentMode: .scaleAspectFit).layout(size: Size(w: 1, h: 1), into: boundingBox.size()).move(dx: boundingBox.x / boundingBox.w, dy: boundingBox.y / boundingBox.h)
+                    let absoluteBounds = locus.bounds()
+                    let respectiveBounds = userSpaceLocus.locus.bounds()
+                    let finalSize = Size(w: absoluteBounds.w * respectiveBounds.w,
+                                         h: absoluteBounds.h * respectiveBounds.h)
+                    let scale = ContentLayout.of(contentMode: .scaleToFill).layout(size: respectiveBounds.size(), into: finalSize)
+                    let transform = Transform.identity.move(dx: absoluteBounds.x, dy: absoluteBounds.y).scale(sx: scale.m11, sy: scale.m22)
```

Comment:
When you make `Transform.identity.move` you actually multiply two matrices and as a result gets simple move matrix. So just use `Transform.move` - same result, better performance.